packages/firebase_auth/firebase_auth_platform_interface/test/method_channel_firebase_auth_test.dart

```diff
@@ -209,6 +209,25 @@ void main() {
       );
     });
 
+    test('signInAnonymously with null additionalUserInfo', () async {
+      final PlatformAuthResult result = await auth.signInAnonymously(appName);
+      MethodChannelFirebaseAuth.channel
+          .setMockMethodCallHandler((MethodCall call) async {
```

Comment:
Does it matter that you're running the code before you set the mock?

Reply:
YES! Oops this is basically the same test as the one before.

Reply:
Fixed.